src/Symfony/Component/HttpFoundation/Cookie.php

```diff
@@ -220,6 +240,39 @@ public function getExpiresTime()
         return $this->expire;
     }
 
+    /**
+     * Creates a cookie copy with a new time the cookie expires.
+     *
+     * @param int|string|\DateTimeInterface $expire
+     *
+     * @throws \InvalidArgumentException
+     */
+    public function withExpiresTime($expire = 0): self
```

Comment:
the argument should be mandatory
withExpires() as a name?

Reply:
I renamed the method, but it cannot be mandatory (see my message above about constructor)